File: C++/2342 - Overflow.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    long a,b,r,s;
    string op;

    cin>>s>>a>>op>>b;

    if(op=="*"){
        r = a*b;
    }
    else{
        r = a+b;
    }

    if(s<r){
        cout<<"OVERFLOW"<<endl;
    }
    else{
        cout<<"OK"<<endl;
    }
}

File: PYTHON/2342 - Overflow.py
n = int(input())

p,c,q = map(str,input().split())

p,q = int(p),int(q)

if (c == "+"):
    if p+q > n:
        print("OVERFLOW")
    else:
        print("OK")

elif c == "*":
    if p*q > n:
        print("OVERFLOW")
    else:
        print("OK")